### PR TITLE
HPCC-20608 Too much logging at log level 1

### PR DESCRIPTION
**Pull request by gaojx**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by gaojx** [comment]
@rpastrana please code review. There are simple changes. I also added one change that add header files to esphttp project to help developer open them conveniently. Since the changes does not affect runtime, and only for Visual Studio, it does worth to have a separate ticket for it.

**Comment by richardkchapman** [comment]
@gaojx Unrelated changes need to be split into separate pull requests. It's always the "so simple that we don't need to follow the normal rules" changes that end up biting us...

**Comment by AttilaVamos** [comment]
The problem with ESP server start is:
```
0000007E 2018-10-08 16:52:40.334  8724  8724 "Warning: Could not load libws_sql.so: libws_sql.so: cannot open shared object file: No such file or directory"
0000007F 2018-10-08 16:52:40.334  8724  8724 "ERROR: Failed to load shared object (libws_sql.so)"
00000080 2018-10-08 16:52:40.334  8724  8724 "ERROR: ESP Unhandled IException (-1 -- procedure esp_service_factory can't be loaded)"
```

File: CMakeLists.txt [hunk 1]
```diff
@@ -166,6 +166,7 @@ if ( PLUGIN )
     HPCC_ADD_SUBDIRECTORY (plugins/spark "SPARK")
 elseif ( NOT MAKE_DOCS_ONLY )
     HPCC_ADD_SUBDIRECTORY (system)
+    HPCC_ADD_SUBDIRECTORY (cmake_modules)
```
Comment (rpastrana) [inline CMakeLists.txt]: I'm not sure how the cmake changes are related to the log level output

File: cmake_modules/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#####################################################
```
Comment (rpastrana) [inline cmake_modules/CMakeLists.txt]: We might need a copyright header here. Also what's the purpose of this file?

File: cmake_modules/shares.cmake [hunk 3]
```diff
@@ -0,0 +1,51 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems�.
```
Comment (rpastrana) [inline cmake_modules/shares.cmake]: new files require current year in copyright header

File: esp/services/CMakeLists.txt [hunk 4]
```diff
@@ -32,7 +36,11 @@ HPCC_ADD_SUBDIRECTORY (WsDeploy "PLATFORM")
 HPCC_ADD_SUBDIRECTORY (ws_packageprocess "PLATFORM")
 HPCC_ADD_SUBDIRECTORY (ws_esdlconfig "PLATFORM")
 HPCC_ADD_SUBDIRECTORY (esdl_svc_engine "PLATFORM")
+
+IF (BUILD_WSSQL)
```
Comment (rpastrana) [inline esp/services/CMakeLists.txt]: There might already be logic in place for this, take a look at WSSQL_SERVICE

**Review by rpastrana — CHANGES_REQUESTED** [review]
@gaojx a few comments...

File: esp/services/CMakeLists.txt [hunk 5]
```diff
@@ -18,8 +18,12 @@ IF (USE_OPENLDAP)
     HPCC_ADD_SUBDIRECTORY (ws_access "PLATFORM")
 ENDIF(USE_OPENLDAP)
 HPCC_ADD_SUBDIRECTORY (ws_account "PLATFORM")
-HPCC_ADD_SUBDIRECTORY (ws_config "PLATFORM")
-HPCC_ADD_SUBDIRECTORY (ws_configmgr "PLATFORM")
+
+IF (NOT WIN32)
```
[the remark this answers is not in the document]
Comment (gaojx) [reply esp/services/CMakeLists.txt]: Yes, the projects do not build on Windows for me. But I agree the right way is to fix the build so I will revert this change.

File: CMakeLists.txt (hunk 1 above)
Comment (gaojx) [reply CMakeLists.txt]: Ok. will put the change into separate ticket.

File: cmake_modules/shares.cmake (hunk 3 above)
Comment (gaojx) [reply cmake_modules/shares.cmake]: added

**Review by rpastrana — CHANGES_REQUESTED** [review]
@gaojx a few more comments

File: esp/services/CMakeLists.txt [hunk 6]
```diff
@@ -1,4 +1,4 @@
-################################################################################
+﻿################################################################################
```
[the remark this answers is not in the document]
Comment (gaojx) [reply esp/services/CMakeLists.txt]: reverted

**Review by rpastrana — APPROVED** [review]
@gaojx looks fine. however esp/services/CMakeLists.txt still shows a change (you might need to unstage that file).

**Comment by gaojx** [comment]
All comments are resolved. @richardkchapman Please merge to 7.0.x

**Pull request merged**